src/components/landing/IdentityCard.tsx

```diff
@@ -58,9 +57,14 @@ export default function ({
           )}
           {left && <NoisyRectangle bgColor="bg-accent" />}
         </div>
-        <BodyText small={xs} center>
-          {text}
-        </BodyText>
+        <span className={displayFromXsToSm}>
```

Comment:
Maybe it's better to have breakpoints here, we're `WET` here

Reply:
As you can see, here the difference is only one prop

Reply:
Changed `span` to `div` but leave it as it is for now

Reply:
Returned it back as you suggested